dist/profile/templates/confluence/vhost.conf

```diff
@@ -3038,6 +3038,12 @@ RewriteRule "^/display/JENKINS/Running\+Jenkins\+behind\+HAProxy$" "https://www.
 RewriteRule "^/display/JENKINS/Running\+Jenkins\+behind\+Squid$" "https://www.jenkins.io/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/#running-jenkins-behind-squid" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/Running\+Jenkins\+behind\+IIS$" "https://www.jenkins.io/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/#running-jenkins-behind-iis" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/Running\+Jenkins\+behind\+Nginx$" "https://www.jenkins.io/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/#running-jenkins-behind-nginx" [NE,NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Executor\+Starvation$" "https://www.jenkins.io/doc/book/using/executor-starvation" [NE,NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Configuring\+Content\+Security\+Policy$" "https://www.jenkins.io/doc/book/system-administration/security/configuring-content-security-policy" [NE,NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Monitor\+and\+Restart\+Offline\+Slaves$" "https://www.jenkins.io/doc/book/managing/nodes" [NE,NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/How\+to\+view\+Jenkins\+in\+your\+language$" "https://www.jenkins.io/doc/book/using" [NE,NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/JenkinsBehindProxy$" "https://www.jenkins.io/doc/book/system-administration/reverse-proxy-configuration-with-jenkins" [NE,NC,L,QSA,R=301]
```

Comment:
JenkinBehindProxy is not the same as Jenkins behind a reverse proxy.  Jenkins behind a proxy is describing the case where a user is insulated from the public internet by a proxy server that acts on their behalf to access the public internet.  The reverse proxy cases use an HTTP / HTTPS server to host, protect, and defend a Jenkins server.
```suggestion
```